frontend/pages/create-a-form.js

```diff
@@ -0,0 +1,128 @@
+import Head from 'next/head'
+import Title from '../components/atoms/Title'
+import Hero from '../components/Hero'
+import DefaultLayout from '../layouts'
+import Button from '../components/atoms/Button'
+import FormCard from '../components/atoms/FormCard'
+import { useContext, useEffect, useReducer } from 'react'
+import SVG from 'react-inlinesvg'
+import UserContext from '../components/context/UserContext'
+import { Tabs, TabList, Tab, TabPanels, TabPanel } from "@reach/tabs";
+import classNames from 'classnames'
+import InputFile from '../components/atoms/InputFile'
+
+const types = {
+  SET_ICON: "SET_ICON",
+  SET_BANNER: "SET_BANNER",
+}
+
+const initialState = {
+  icon: [],
+  banner: [],
+}
+
+function reducer(state, action) {
+  switch (action.type) {
+    case types.SET_ICON:
+      return {
+        ...state,
+        icon: action.value,
+      }
+    case types.SET_BANNER:
+      return {
+        ...state,
+        banner: action.value,
+      }
+    default:
+      return state
+  }
+};
+
+export default function Dashboard() {
+  const { user } = useContext(UserContext)
+  const [state, dispatch] = useReducer(reducer, initialState)
+
+  if (!user) {
+    return (
```

Comment:
Might be better to split an unauthorized page into a separate component since we have more than one page that requires auth

Reply:
You are right, on it